lerna.json

```diff
@@ -17,7 +17,7 @@
   "packages": [
     "lib/*",
     "experimental/*",
-    "samples/**/*"
+    "samples/**/**/*"
```

Comment:
shouldn't be needed

Reply:
Yep, you're right, reverted this change.